debian_server/privacyidea-apache2.postinst

```diff
@@ -95,7 +95,7 @@ create_database() {
 	    NPW="$(tr -dc A-Za-z0-9_ </dev/urandom | head -c12)"
 	    mysql -u $USER --password=$PASSWORD -e "create database pi;" || true
 	    mysql -u $USER --password=$PASSWORD -e "grant all privileges on pi.* to 'pi'@'localhost' identified by '$NPW';"
-	    echo "SQLALCHEMY_DATABASE_URI = 'mysql+pymysql://pi:$NPW@localhost/pi'" >> /etc/privacyidea/pi.cfg
+	    echo "SQLALCHEMY_DATABASE_URI = 'mysql+pymysql://pi:$NPW@localhost/pi?charset=utf8mb4'" >> /etc/privacyidea/pi.cfg
```

Comment:
I did some digging to find out why this is necessary for the Audit SQL connection, and not for our "main" SQL connection. The reason is that the main SQL connection uses flask-sqlalchemy, which sets the ``charset`` parameter to ``utf8`` in case of mysql:

https://github.com/pallets/flask-sqlalchemy/blob/500e732dd1b975a56ab06a46bd1a20a21e682262/flask_sqlalchemy/__init__.py#L835

And our Audit connection doesn't use flask-sqlalchemy, which is why it doesn't automatically set that parameter.

So: I know I suggested ``utf8mb4`` myself, but can we use ``utf8`` instead? :-) This would be consistent with the flask-sqlalchemy parameter.

Reply:
done in 5091c7d

Reply:
Thx!